src/main/java/seedu/address/logic/commands/session/AddSessionCommand.java

```diff
@@ -31,7 +31,8 @@
             + PREFIX_DURATION + "120 ";
 
     public static final String MESSAGE_SUCCESS = "New Session added: %1$s";
-    public static final String MESSAGE_DUPLICATE_SESSION = "This Session already exists in FitEgo.";
+    public static final String MESSAGE_DUPLICATE_SESSION = "This Session overlaps with an existing Session in FitEgo.\n"
+            + "A Session is considered overlapping if another Session starts before the current session ends.";
```

Comment:
should the second session start with capital s?

Reply:
right!